keyboards/C64/config.h

```diff
@@ -0,0 +1,156 @@
+/*
+Copyright 2012 Jun Wako <wakojun@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+/* Keyboard Firmware for TheC64mini by retrogames. This firmware works on the Upgrade Keyboard Kit sold by Dean Woodyatt at Bleugh.biz */
+
+#pragma once
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x6060
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Bleugh
+#define PRODUCT         C64_Mini_Keyboard
+#define DESCRIPTION     commodore_mini
+
+/* key matrix size */
+#define MATRIX_ROWS 9
+#define MATRIX_COLS 9
+
+/*
+ * Keyboard Matrix Assignments
+ *
+ * Change this to how you wired your keyboard
+ * COLS: AVR pins used for columns, left to right
+ * ROWS: AVR pins used for rows, top to bottom
+ * DIODE_DIRECTION: COL2ROW = COL = Anode (+), ROW = Cathode (-, marked on diode)
+ *                  ROW2COL = ROW = Anode (+), COL = Cathode (-, marked on diode)
+ *
+*/
+//#define MATRIX_ROW_PINS { 9, 3, 4, 5, 6, 7, 8, 2,0 }			    // ARDUINO PINS THAT WORK
+//#define MATRIX_COL_PINS { 10, 16, 14, 21, 18, 19, 20, 15, 1}      // ARDUINO PINS THAT WORK
```

Comment:
```suggestion
```

Also, for reference: https://cdn.sparkfun.com/datasheets/Dev/Arduino/Boards/ProMicro16MHzv1.pdf

Even if you're not using a pro micro, this lists both the Arduino pins, and the "port" pins that QMK uses (drop the "P" in the name